browser/static/js/main_track.js

```diff
@@ -545,72 +544,70 @@ function label_under_mouse() {
 }
 
 function render_highlight_info() {
+  const highlightText = (current_highlight) ? 'ON' : 'OFF';
+  let currentlyHighlighted = 'none';
   if (current_highlight) {
-    $('#highlight').html("ON");
-    if (mode.highlighted_cell_one !== -1) {
-      if (mode.highlighted_cell_two !== -1) {
-        $('#currently_highlighted').html(mode.highlighted_cell_one + " , " + mode.highlighted_cell_two);
-      } else {
-        $('#currently_highlighted').html(mode.highlighted_cell_one);
-      }
+    if (edit_mode) {
+      currentlyHighlighted = (brush.value > 0) ? brush.value : '-';
     } else {
-      $('#currently_highlighted').html("none");
+      if (mode.highlighted_cell_one !== -1) {
+        if (mode.highlighted_cell_two !== -1) {
+          currentlyHighlighted = `${mode.highlighted_cell_one}, ${mode.highlighted_cell_two}`;
+        } else {
+          currentlyHighlighted = mode.highlighted_cell_one;
+        }
+      }
     }
-  } else {
-    $('#highlight').html("OFF");
-    $('#currently_highlighted').html("none");
   }
+  document.getElementById('highlight').innerHTML = highlightText;
+  document.getElementById('currently_highlighted').innerHTML = currentlyHighlighted;
 }
 
 function render_edit_info() {
-  if (edit_mode) {
-    $('#edit_mode').html('pixels');
-    $('#edit_brush_row').css('visibility', 'visible');
-    $('#edit_label_row').css('visibility', 'visible');
-    $('#edit_erase_row').css('visibility', 'visible');
+  const editModeText = (edit_mode) ? 'paint mode' : 'whole-label mode';
+  document.getElementById('edit_mode').innerHTML = editModeText;
 
-    $('#edit_brush').html(brush.size);
-    $('#edit_label').html(brush.value);
+  const rowVisibility = (edit_mode) ? 'visible' : 'hidden';
+  document.getElementById('edit_brush_row').style.visibility = rowVisibility;
+  document.getElementById('brush_label_row').style.visibility = rowVisibility;
+  document.getElementById('edit_erase_row').style.visibility = rowVisibility;
 
-    if (brush.erase) {
-      $('#edit_erase').html("ON");
-    } else {
-      $('#edit_erase').html("OFF");
-    }
+  if (edit_mode) {
+    document.getElementById('edit_brush').innerHTML = brush.size;
 
-  } else {
-    $('#edit_mode').html('whole labels');
-    $('#edit_brush_row').css('visibility', 'hidden');
-    $('#edit_label_row').css('visibility', 'hidden');
-    $('#edit_erase_row').css('visibility', 'hidden');
+    const editLabelText = (brush.value > 0) ? brush.value : '-';
+    document.getElementById('brush_label').innerHTML = editLabelText;
+
+    const editEraseText = (brush.erase && !brush.conv) ? 'ON' : 'OFF';
```

Comment:
I'm just flagging this because the logic looks a little different - I want to confirm the behavior here.

Reply:
With line 578, the new version sets the brush label text to '-' when the value is negative. I think that a brush label of -1 is supposed to represent no label, but I can't anywhere in brush.js, main_track.js, or main_zstack.js that sets a negative brush label...

With lines 579, we changed the row name from 'edit_label' to 'brush_label', so that should be right.

With line 581, we can't turn on the conversion brush in the tracking tool, so brush.conv will always be false, so the behavior is unchanged.

Overall, these changes make the function nearly identical to the `main_zstack.js`, so I think it's worth keeping the complexity to make it easy to merge their code in the future.